src/views/ArticleTags.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Tags() {
+  return (
+    <div className="single-tags text-center">
+      <span className="badge">andela</span>
```

Comment:
#### Nitpick
- We could make this enumerable
```
const badges = ['technology', 'bootcamp', 'computer'];

...............


badges.map(badge => {
return (
<span className="badge">badge</span>
)
})

```

Reply:
The reason the categories and the other things like taglist aren't handled yet is because I need to include them in the backend. it's not part of the response. So I'll be making provision for that tomorrow. 

I'll be explaining the details of what I need to do in the TTL call tomorrow.

Reply:
@thislekan , You will still be working with an array of badges from the backend, right ?

Reply:
Yes, I will. I just left this part because Valentine needs the view and I think there are so many other task dependent on it. These tasks also need the stuff I will be working on in the backend.

Reply:
Okay understood